.gitignore

```diff
@@ -0,0 +1,3 @@
+manual_test.py
+__pycache__/
+*.tokens-app
```

Comment:
The first and last of these are presumably your own files, and aren't relevant to the library itself.  `__pycache__` is valid, but better handled system-wide with a personal gitignore file on your own machine.